src/components/LineChart/LineChart.tsx

```diff
@@ -32,21 +32,22 @@ export class LineChart extends React.PureComponent<ILineChartProps, any> {
         this.state = {
             width: 0,
             height: 0,
-            isParentMounted: false
+            isParentMounted: false,
+            chartId: 'line-' + Guid.raw()
         };
     }
 
     public render() {
-        const props = objectAssign({}, this.props, {width: this.state.width, height: this.state.height});
-        const componentClass = classNames('line-chart-component', this.props.id, this.props.className);
-        const titleClass = classNames('line-chart-title', this.props.id, this.props.className);
+        const props = objectAssign({}, this.props, { width: this.state.width, height: this.state.height, id: this.state.chartId });
```

Comment:
You can remove the ID property out of the props if you are assigning the id from the state to props. This why you are basically changing the props in the render method